packages/dashboard/src/pages/Settings/api.ts

```diff
@@ -43,6 +43,11 @@ export const [useCeloNetworkTradeProvider] = createGlobalState(
     Messages.events.celoNetworkTradeProviderSettings.on,
 )
 
+export const [useOpNetworkTradeProvider] = createGlobalState(
```

Comment:
`useXXTradeProvider` will be removed. see https://github.com/DimensionDev/Maskbook/pull/5516

Reply:
done